app/websockets.py

```
@@ -21,36 +21,40 @@
     os.makedirs(ready_dir, exist_ok=True)
-    for client_id, details in MEETING_DETAILS.items():
-        if len(details) > 0 and floor_key(details[0]) == key:
-            try:
-                with open(os.path.join(ready_dir, f"{client_id}.ready"), "w") as f:
-                    f.write(datetime.now().isoformat())
-                logger.info(f"Roster seen for meeting {key} — ready signal for {client_id}")
-            except OSError as e:
-                logger.warning(f"Could not write ready signal for {client_id}: {e}")
+    try:
+        with open(os.path.join(ready_dir, f"{client_id}.ready"), "w") as f:
+            f.write(datetime.now().isoformat())
```

Ruff (0.15.21)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)